connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/correlation/InboundCorrelationHandler.java

```diff
@@ -146,13 +149,41 @@ protected boolean isActivationConditionMet(
       return true;
     }
     try {
+      // quick fix cases when activation condition with '-' like
+      // '=(request.body.detail-type="MyEvent")'
```

Comment:
I think, we should not provide fixes on such cases. It seems to be FEEL engine specifics.

Customers could use `=(get value(request.body, "detail-type") = "MyEvent")` instead.